app/v1/models/errors/mtdErrors.scala

```diff
@@ -108,3 +108,14 @@ object RuleDateRangeTooLargeError extends MtdError(
 object InvalidDateToErrorDes extends MtdError("DATE_TO_INVALID", "The provided to date is invalid")
 
 object InvalidDateFromErrorDes extends MtdError("DATE_FROM_INVALID", "The provided from date is invalid")
+
+object InvalidMonetaryValueError extends MtdError("INVALID_REQUEST", "Invalid request") {
```

Comment:
```suggestion
object InvalidMonetaryAmountError extends MtdError("INVALID_REQUEST", "Invalid request") {
```

Also, the message should be different depending on the specific validation rules of the particular field which is incorrect.